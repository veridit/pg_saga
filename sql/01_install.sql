/* Once support for 9.5 has passed, use CASCADE */
CREATE EXTENSION IF NOT EXISTS btree_gist;
/* Once support for 9.6 has passed, just create the extension */
CREATE EXTENSION periods VERSION '1.2';

SELECT extversion
FROM pg_extension
WHERE extname = 'periods';

DROP ROLE periods_unprivileged_user;
CREATE ROLE periods_unprivileged_user;

/* Make tests work on PG 15 */
GRANT CREATE ON SCHEMA public TO PUBLIC;
