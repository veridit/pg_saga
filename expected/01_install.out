/* Once support for 9.5 has passed, use CASCADE */
CREATE EXTENSION IF NOT EXISTS btree_gist;
/* Once support for 9.6 has passed, just create the extension */
CREATE EXTENSION periods VERSION '1.2';
SELECT extversion
FROM pg_extension
WHERE extname = 'periods';
 extversion 
------------
 1.2
(1 row)

DROP ROLE periods_unprivileged_user;
ERROR:  role "periods_unprivileged_user" does not exist
CREATE ROLE periods_unprivileged_user;
/* Make tests work on PG 15 */
GRANT CREATE ON SCHEMA public TO PUBLIC;
